src/TTImp/Elab/RunElab.idr

```diff
@@ -228,6 +259,22 @@ elabScript rig fc nest env script@(NDCon nfc nm t ar args) exp
              decls <- reify defs d'
              traverse_ (processDecl [] (MkNested []) []) decls
              scriptRet ()
+    elabCon defs "ReadFile" [lk, pth]
+        = do pathPrefix <- lookupDir defs !(evalClosure defs lk)
+             path <- reify defs !(evalClosure defs pth)
+             let fullPath = joinPath [pathPrefix, path]
```

Comment:
Should we not make sure that `path`  does not contain `..`?

Reply:
Yes, I think we should, I'll do this

Reply:
I did it (and minor another thing) and added a test for newly added stuff to the `LessSimpleRW.idr` to the same test